File: SimpleInterest.py
def simpleInterest(p, t, r):
    return (p * t * r) / 100

# Nhập và chuyển đổi đầu vào
p, t, r = map(int, input("Hãy nhập lần lượt các hệ số p, t, r cách nhau bởi khoảng trắng: ").split())

# Gọi hàm và in kết quả
print(f"Lãi suất đơn là: {simpleInterest(p, t, r):.2f}")

File: SimpleInterest.c
#include <stdio.h>

float simpleInterest(float p, float t, float r) {
    float ans = (p * t * r) / 100;
    return ans;
}

int main() {
    float p, t, r;
    printf("Nhập các hệ số p, t, r cách nhau bởi khoảng trắng: ");
    scanf("%f %f %f", &p, &t, &r);
    printf("%.2f", simpleInterest(p, r, t));
    return 0;
}
